ts/components/messages/MessageDetailRawInfoComponent.tsx

```diff
@@ -60,14 +62,14 @@ class MessageDetailRawInfoComponent extends React.PureComponent<Props, State> {
     if (isOpen) {
       return (
         <View>
-          <TouchableWithoutFeedback onPress={this.toggleRawInfo}>
+          <TouchableWithoutOpacity onPress={this.toggleRawInfo}>
             <View style={styles.toggleContainer}>
-              <Text link={true}>
+              <Text style={styles.linkStyle}>
```

Comment:
I guess this style is no needed since it alters the font in bold but originally it is not bold
you changes underlined in red
<img src="https://user-images.githubusercontent.com/822471/68233443-82571800-ffff-11e9-816b-e2854eea5a35.png" width="800"/>

Reply:
i remove the style and use link=true